src/Framework/App.Ref/src/Microsoft.AspNetCore.App.Ref.csproj

```diff
@@ -259,30 +258,18 @@ This package is an internal implementation of the .NET Core SDK and is not meant
   </Target>
 
   <Target Name="_CreateTargetingPackArchive"
-          Condition=" '$(IsPackable)' == 'true' "
           Inputs="@(RefPackContent)"
-          Outputs="$(ZipArchiveOutputPath);$(TarArchiveOutputPath)">
-    <PropertyGroup>
-      <_TarCommand>tar</_TarCommand>
-      <_TarCommand Condition="Exists('$(RepoRoot).tools\tar.exe')">"$(RepoRoot).tools\tar.exe"</_TarCommand>
-
-      <!-- For the tar packed with git, transform e.g. "C:\root\AspNetCore\File.tar.gz" to "/C/root/AspNetCore/File.tar.gz". -->
-      <_TarArchiveOutputPath>$(TarArchiveOutputPath)</_TarArchiveOutputPath>
-      <_TarArchiveOutputPath
-          Condition="Exists('$(repoRoot)\.tools\tar.fromGit')">/$(TarArchiveOutputPath.Replace('\','/').Replace(':',''))</_TarArchiveOutputPath>
-    </PropertyGroup>
```

Comment:
If we don't need this anymore, remove eng/scripts/InstallTar.ps1, its use in default-build.yml, and update the build from source docs.